Self Patch/Patch.cs

```diff
@@ -65,19 +66,19 @@ private void backgroundWorker1_RunWorkerCompleted(object sender, RunWorkerComple
 
         private void CheckConnectivity()
 		{
-            if (this.settings.RemotePatchLocation.Contains("discoverygc.com"))
-            {
-                this.settings.RemotePatchLocation = "http://patch.discoverygc.net/";
-            }
+			if (this.settings.RemotePatchLocation.Contains("discoverygc.com"))
+			{
+				this.settings.RemotePatchLocation = "http://patch.discoverygc.net/";
+			}
 
-            try
+			try
```

Comment:
What's going on with the indentations in this file?

Reply:
I didn't make any changes there.
Seems my IDE replace spaces by tabs...
I will restore them later...